apps/web/playwright/webhook.e2e.ts

```diff
@@ -557,6 +536,17 @@ test.describe("BOOKING_RESCHEDULED", async () => {
   });
 });
 
+/**
+ * TO-DO:
+ * Implement these tests using the examples on apps/web/playwright/integrations-stripe.e2e.ts
+ * We need to do something agnostic to the payment provider here, so we can test it with Stripe, Paypal or other providers
```

Comment:
Is someone working on this agnostic layer already?

This could be a tremendous facilitator in terms of scaling the product. We can have dozens of providers a few months from now so if we have this layer, it'd be beneficial from a technical perspective.

If someone is already working on it, we can open a new issue to add these tests. If no one is working on it, I'd be happy to help